processor/src/main/java/org/mapstruct/ap/internal/conversion/JavaLocalDateToSqlDateConversion.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.internal.conversion;
+
+import java.sql.Date;
+import java.util.Set;
+
+import org.mapstruct.ap.internal.model.common.ConversionContext;
+import org.mapstruct.ap.internal.model.common.Type;
+import org.mapstruct.ap.internal.util.Collections;
+
+import static org.mapstruct.ap.internal.conversion.ConversionUtils.sqlDate;
+import static org.mapstruct.ap.internal.conversion.ConversionUtils.zoneOffset;
+import static org.mapstruct.ap.internal.util.JavaTimeConstants.ZONE_OFFSET_FQN;
+
+/**
+ * SimpleConversion for mapping {@link java.time.LocalDate} to
+ * {@link Date} and vice versa.
+ */
+public class JavaLocalDateToSqlDateConversion extends SimpleConversion {
```

Comment:
You can add your name as `@author` if you want to

Reply:
I don't want to add my name as an author yet. This is my first PR contribution to open source. Thanks a lot for helping me out.

I hope to do more so I can add my name as an author at some point later 😀.

Reply:
Thanks for letting us know. It's your choice. Really glad that you decided to contribute to MapStruct as your first contribution.

Would it be OK for you if we add your name to https://github.com/mapstruct/mapstruct/blob/master/copyright.txt?